spec/controllers/rooms_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe RoomsController, type: :controller do
```

Comment:
I, personally, prefer to use `request` type for controller specs so the views are rendered, etc